src/map/script-fun.cpp

```diff
@@ -3117,6 +3117,32 @@ BuiltinFunction builtin_functions[] =
     BUILTIN(getmap, ""_s, 's'),
     BUILTIN(mapexit, ""_s, '\0'),
     BUILTIN(freeloop, "i"_s, '\0'),
+    // Magic builtins needed
+    //BUILTIN(sfx, ".ii"_s),
+    //BUILTIN(instaheal, "eii"_s),
+    //BUILTIN(itemheal, "eii"_s),
+    //BUILTIN(shroud, "ei"_s),
+    //BUILTIN("unshroud"_s, "e"_s, op_reveal),
+    //BUILTIN(message, "es"_s),
+    //BUILTIN(messenger_npc, "lissi"_s),
+    //BUILTIN(move, "ed"_s),
+    //BUILTIN(warp, "el"_s),
+    //BUILTIN(banish, "e"_s),
+    //BUILTIN(status_change, "eiiiiii"_s),
+    //BUILTIN(stop_status_change, "ei"_s),
+    //BUILTIN(override_attack, "eiiiiii"_s),
+    //BUILTIN(create_item, "e.i"_s),
+    //BUILTIN(aggravate, "eie"_s),
+    //BUILTIN(spawn, "aeiiii"_s),
+    //BUILTIN(injure, "eeii"_s),
+    //BUILTIN(emote, "ei"_s),
+    //BUILTIN(set_script_variable, "esi"_s),
+    //BUILTIN(set_script_str, "ess"_s),
+    //BUILTIN(set_hair_colour, "ei"_s),
+    //BUILTIN(set_hair_style, "ei"_s),
+    //BUILTIN("drop_item"_s, "l.ii"_s, op_drop_item_for),
+    //BUILTIN(drop_item_for, "l.iiei"_s),
```

Comment:
Also I think
+    //BUILTIN(emote, "ei"_s),
+    //BUILTIN(set_script_variable, "esi"_s),
+    //BUILTIN(set_script_str, "ess"_s),
+    //BUILTIN(set_hair_colour, "ei"_s),
+    //BUILTIN(set_hair_style, "ei"_s),
Can use the NPC ones or other better methods.